src/admin/forms/BannerForm.js

```diff
@@ -0,0 +1,217 @@
+import React, {useState} from 'react';
+import {withRouter} from 'react-router-dom';
+import {Button, Form, Message} from 'semantic-ui-react';
+
+const SEVERITY = [
+  {key: 'INFO', name: 'Info'},
+  {key: 'WARN', name: 'Warning'},
+  {key: 'ERROR', name: 'Error'},
+];
+
+const BannerForm = ({history, isUpdate, mutation, banner, loading, err}) => {
+  const [submitting, setSubmitting] = useState();
+  const [error, setError] = useState(err);
+  const [state, setState] = useState({
```

Comment:
This looks like it will work, although we tend to use `Formik` to set up all and store all the state, submission, and validation for us. Perhaps you want to try wrapping your form with it to see how it works?